geomstats/numerics/geodesic.py

```diff
@@ -62,26 +71,33 @@ class ExpODESolver(ExpSolver):
     """
 
     def __init__(self, integrator=None):
+        super().__init__(solves_ivp=False)
+
         if integrator is None:
             integrator = GSIVPIntegrator()
 
+        self._integrator = None
```

Comment:
what's the difference between _integrator and integrator? Could you add the description of the parameters to help readability?

Reply:
This is a `Python` thing. We want to make an action when doing `solver.integrator = SomeIntegrator()`. I'll check if I can simplify this part to avoid confusion.